lib/components/user/trip-basics-pane.js

```diff
@@ -31,73 +37,142 @@ const StyledLabel = styled.label`
-        </FormGroup>
-      </div>
-    )
+
+      // Show a combined error indicaton when no day is selected.
+      let monitoredDaysValidationState = null
+      ALL_DAYS.forEach(day => {
+        if (touched[day]) {
+          if (!monitoredDaysValidationState) {
+            monitoredDaysValidationState = errors[day] ? 'error' : null
+          }
+        }
+      })
+
+      return (
+        <div>
+          <ControlLabel>Selected itinerary:</ControlLabel>
+          <TripSummary monitoredTrip={monitoredTrip} />
+
+          <FormGroup validationState={tripNameValidationState}>
+            <ControlLabel>Please provide a name for this trip:</ControlLabel>
+            {/* onBlur, onChange, and value are passed automatically. */}
+            <Field as={FormControl} name='tripName' />
+            <FormControl.Feedback />
+            {errors.tripName && <HelpBlock>{errors.tripName}</HelpBlock>}
+          </FormGroup>
+
+          <FormGroup validationState={monitoredDaysValidationState}>
+            <ControlLabel>What days do you take this trip?</ControlLabel>
+            <div>
+              {allDays.map(({ name, fullText, text }, index) => {
+                const isDayDisabled = itineraryExistence && !itineraryExists(itineraryExistence[name])
+                const boxClass = isDayDisabled ? 'alert-danger' : (monitoredTrip[name] ? 'bg-primary' : '')
+                const notAvailableText = isDayDisabled ? `Trip not available on ${fullText}` : null
+
+                return (
+                  <TripDayLabel className={boxClass} key={index} title={notAvailableText}>
+                    <span>{text}</span>
+                    { // Let users save an existing trip, even though it may not be available on some days.
+                      // TODO: improve checking trip availability.
+                      isDayDisabled && isCreating
+                        ? <Glyphicon aria-label={notAvailableText} glyph='ban-circle' />
+                        : <Field name={name} type='checkbox' />
+                    }
+                  </TripDayLabel>
+                )
+              })}
+              <div style={{clear: 'both'}} />
+            </div>
+            <HelpBlock>
+              {itineraryExistence
+                ? (
+                  <>Your trip is available on the days of the week as indicated above.</>
+                ) : (
+                  <ProgressBar active label='Checking itinerary existence for each day of the week...' now={100} />
+                )
+              }
+            </HelpBlock>
+            {monitoredDaysValidationState && <HelpBlock>Please select at least one day to monitor.</HelpBlock>}
+          </FormGroup>
+        </div>
+      )
+    }
+  }
+}
+
+// Connect to redux store
+const mapStateToProps = (state, ownProps) => {
+  const { accessToken, itineraryExistence } = state.user
+  return {
+    accessToken,
```

Comment:
Is `accessToken` needed still?

Reply:
Good find. Updated in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/846892ab816dc32ccd6688b06042f030225c453a.